includes/class-sensei.php

```diff
@@ -686,6 +686,10 @@ public function activate_sensei() {
 			update_option( 'sensei_show_email_signup_form', true );
 		}
 
+		if ( false === get_option( Sensei_Onboarding::SUGGEST_ONBOARDING_OPTION, false ) ) {
+			update_option( Sensei_Onboarding::SUGGEST_ONBOARDING_OPTION, 1 );
```

Comment:
I think we only want to show the wizard notice after the first installation, and not for existing users updating or re-activating the plugin - is that right @donnapep?
If that is the case, this can go a few lines higher, added only when `sensei_installed` is not set.
(Though that would make testing this harder, so maybe we can delay it to a future PR.)

Reply:
Yes, that's correct. Existing users are already onboarded. 🙂

Reply:
Thank you for the suggestion, @yscik. It's updated here: https://github.com/Automattic/sensei/pull/3096/commits/dde8a08d186981588373bad1643f96941751b6c9